src/test/java/org/alfresco/repo/search/impl/solr/TestNode.java

```diff
@@ -131,12 +131,18 @@ public TransactionEntity getTransaction()
     public AuditablePropertiesEntity getAuditableProperties()
     {
         return null;
-    }
-
-    @Override
-    public String getShardKey()
-    {
-        return null;
+    }
+
+    @Override
+    public String getShardKey()
+    {
+        return null;
+    }
+
+    @Override
+    public Integer getExplicitShardId()
+    {
+        return null;

```

Comment:
Only including "getExplicitShardId" method, but changes are affecting other parts of the code.